description.ext

```diff
@@ -1,17 +1,20 @@
 author = "AUTHOR NAME";
-OnLoadName = "Operation Name";
-OnLoadMission = "Loading text";
+OnLoadName = "MISSION NAME";
+OnLoadMission = "MISSION DESCRIPTION";
 loadScreen = "load.jpg";
 
+enableDebugConsole[] = {"76561199020930438"};
+
 // respawnTemplates[] = {"TCS_OneLife"}; // One life respawn
-// respawnTemplates[] = {"TCS_Respawn", "TCS_Tickets"}; // Ticket based respawn
-// respawnTemplates[] = {"TCS_Respawn", "TCS_Individual_Tickets"}; // Individual ticket based respawn
-respawnTemplates[] = {"TCS_Respawn", "Counter"}; // Default respawn
+ respawnTemplates[] = {"TCS_Respawn", "TCS_Tickets"}; // Ticket based respawn
+//respawnTemplates[] = {"TCS_Respawn", "TCS_Individual_Tickets"}; // Individual ticket based respawn
+// respawnTemplates[] = {"TCS_Respawn", "Counter"}; // Default respawn

```

Comment:
Do we want to change the default option?

Reply:
No looks like my personal mission got added to source control. I'll fix that